projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -11,6 +11,10 @@
     background: #fafafa;
     border-bottom: 1px solid var(--theme-border-color, rgba(0, 0, 0, 0.07));
     padding: 0 24px;
+
+    .mat-toolbar {
+      color: var(--theme-text-color, rgba(0, 0, 0, 0.54));
```

Comment:
might cause issues for cases when the page layout is not used?

Reply:
true... though we use it everywhere, at the moment.
But, indeed, it would be best to be able to set this color for the ADF toolbar component like we do with the `$adf-toolbar-padding` for example, in `custom-theme.scss` line 85. I'll add the possibility to have a `$adf-toolbar-color` in a PR on ADF.

Reply:
meanwhile, I moved this color setting so it's no longer dependent on using the page layout